ui-core/src/components/inputs/datePicker/Calendar.tsx

```diff
@@ -34,16 +34,13 @@ const Day = ({ date, isReferenceDate, isSelectable, dayWrapperClassName, onClick
   </div>
 );
 
-const Calendar = ({ displayedMonthStartDate, onDayClick }: CalendarProps) => {
-  const { days, isReferenceDate, isDateSelectable, buildDayWrapperClassName } = useCalendar({
-    displayedMonthStartDate,
-    onDayClick,
-  });
+const Calendar = (props: CalendarProps) => {
+  const { days, isReferenceDate, isDateSelectable, buildDayWrapperClassName } = useCalendar(props);
```

Comment:
Can you add a line like this ?
```suggestion
  const { days, isReferenceDate, isDateSelectable, buildDayWrapperClassName } = useCalendar(props);
  const { displayedMonthStartDate, onDayClick } = props;
```